edgelet/edgelet-docker/src/runtime.rs

```diff
@@ -337,6 +337,18 @@ impl ModuleRuntime for DockerModuleRuntime {
                     .unwrap_or_else(HashMap::new);
                 labels.insert(LABEL_KEY.to_string(), LABEL_VALUE.to_string());
 
```

Comment:
Doing this in `create` (which gets called for all containers) isn't great IMO. A better option would have been the watchdog, which has code dedicated solely to creating Edge Agent in the bootstrap scenario. But unfortunately that code doesn't know about Docker; this is the first (or best) place I could find that actually knows about Docker and things like container labels.

Reply:
If anyone has a better idea I'd love to hear it...

Reply:
Just like https://github.com/Azure/iotedge/pull/2305#issuecomment-574785186 , can you not do it in `init_agent_spec` ?

Reply:
🙄 I completely forgot about that PR. Yes, I bet that will work nicely...